src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -309,7 +309,7 @@ public BitrateController(VideoChannel dest)
 
         ConfigurationService cfg = LibJitsi.getConfigurationService();
 
-        trustBwe = cfg != null && cfg.getBoolean(TRUST_BWE_PNAME, false);
+        trustBwe = cfg != null && cfg.getBoolean(TRUST_BWE_PNAME, true);
```

Comment:
Is this intentional? Only asking because the commit message doesn't indicate functional changes